pontoon/base/templates/translate.html

```diff
@@ -326,7 +326,13 @@ <h3 class="loading"><div class="fa fa-refresh fa-spin"></div></h3>
           <button id="clear" title="Clear Translation (Ctrl + Shift + Backspace)">Clear</button>
           <button id="save" title="Save Translation (Enter)"></button>
         {% else %}
-          <p id="sign-in-required"><a id="sidebar-signin" href="{{ provider_login_url(request) }}">Sign in</a> to translate.</p>
+          <p id="sign-in-required">
```

Comment:
ad2. Could you look [here](https://github.com/mozilla/pontoon/blob/master/pontoon/base/static/js/main.js#L485)? There's only a selector missing (or you should change your html these selectors).